src/SWP/Bundle/FixturesBundle/DataFixtures/ORM/LoadWidgetsData.php

```diff
@@ -26,11 +28,14 @@ class LoadWidgetsData extends AbstractFixture implements FixtureInterface, Order
      */
     public function load(ObjectManager $manager)
     {
+        /** @var TenantInterface $tenant */
```

Comment:
That's why we rely on tenant's code instead of relationship. It was changed to PHPCR because we were duplicating the tenants in phpcr. Tenants were stored using doctrine orm and when we initialized phpcr tenants were also inserted (documents) based on the tenants which makes no sense. We kept the tenants in orm and just a tenant document with id in phpcr. Now we keep all in PHPCR and rely on tenant's code when using entities.